lib/components/nav/button_links.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:html';
+
+class Link {
+  String label;
+  String url;
```

Comment:
You have a better feel for how this is to be used, but I'm wondering if it would make sense to have `selected` be an attribute of this class (that's by default set to `false`) instead of passing in the URL of the selected one to the `ButtonLinksView` constructor?

Reply:
I think I will prefer a separate attribute outside of the Link class since there can be only 1 selected 🤔